docs/dynamo.md

```diff
@@ -0,0 +1,90 @@
+## TorchDynamo(torch.compile) integration in PyTorch XLA
+
+Torchdynamo is a Python-level JIT compiler designed to make unmodified PyTorch programs faster. It provides a clean API for compiler backends to hook in and its biggest feature is to dynamically modify Python bytecode right before it is executed. In the pytorch/xla 2.0 release, PyTorch/XLA provided an experimental backend for the TorchDynamo for both inference and training. 
```

Comment:
Can we add this reference to pytorch dynamo - for additional : https://pytorch.org/tutorials/intermediate/dynamo_tutorial.html?